contrib/download-frozen-image-v2.sh

```diff
@@ -65,6 +65,7 @@ fetch_blob() {
 	curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
 	if echo "$curlHeaders" | grep -qE "^HTTP/[0-9].[0-9] 3"; then
```

Comment:
Might be worth also trying this with Bash's `<<<` syntax instead of the `echo | ...`, ala:
```bash
if grep -qE "^HTTP/[0-9].[0-9] 3" <<<"$curlHeaders"; then
```